src/index.d.ts

```diff
@@ -0,0 +1,4 @@
+import { ReactNode } from "react";
+import ImageProps from './components/Image/Image'
+export const ImageComponent: React.ComponentType<ImageProps>;
```

Comment:
Should be `declare const` since this isn't exported in the actual JS implementation

Reply:
Could you check now 👍 thanks mate.